adodb-memcache.lib.inc.php

```diff
@@ -28,17 +28,21 @@
 $db->memCacheHost = array($ip1, $ip2, $ip3);
 $db->memCachePort = 11211; /// this is default memCache port
 $db->memCacheCompress = false; /// Use 'true' to store the item compressed (uses zlib)
+                               /// Note; compression is not supported w/the memcached library
 
 $db->Connect(...);
 $db->CacheExecute($sql);
 
-  Note the memcache class is shared by all connections, is created during the first call to Connect/PConnect.
+  Notes; The memcache class is shared by all connections, is created during the first call to Connect/PConnect.
+         We'll look for both the memcache library (https://pecl.php.net/package/memcache) and the  memcached
+         library (https://pecl.php.net/package/memcached). If both exist, the memcache library will be used.
 
   Class instance is stored in $ADODB_CACHE
 */
 
 	class ADODB_Cache_MemCache {
 		var $createdir = false; // create caching directory structure?
+		var $library = false; // check for memcache/memcached prior to connect, then populate this variable
```

Comment:
This comment is not very clear; when I read it, I have the impression that I need to manually set it. If that's the case, I think it needs to be better documented. But the code in connect() method initializes it...